src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,546 @@
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select your .NET MAUI app) > Summary**: View .NET MAUI data, track HTTP requests and errors, and monitor how your app is performing over time.
+</figcaption>
+
+## Install the .NET MAUI SDK
+<Steps>
+
+<Step>
+### Review the requirements
+
+Before you install our .NET MAUI agent, make sure your app meets these version requirements:
+
+For Android-native apps:
+    * Android 7 (API 24) or higher
+
+For iOS-native apps:
+    * iOS 11 or higher
+    * Latest release of Xcode
+</Step>
+
+<Step>
+### Add the .NET MAUI agent to your project
+
+First, you'll need to add our agent, a NuGet package, to your MAUI project:
+
+1. Open your .NET MAUI solution, select the project you want to add the agent to, and open its context menu.
+2. Click **Add > Add NuGet packages**, then select `NewRelic.MAUI.Plugin`.
+
+</Step>
+
+<Step>
+### Copy your application token from the UI [#app-token]
+
+The application token is used for New Relic to authenticate your Flutter app's data. 
+
+To view and copy your application token in the New Relic UI:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Add data**, then click **Mobile**.
+2. Select your .NET MAUI app. 
+3. Go to **Settings > Application** and copy the displayed **Application token**.
+
+You'll add this application token in the next step.
+</Step>
+
+<Step>
+### Add our agent configuration file to your .NET MAUI project
+
+1. In your project, open `App.xaml.cs` and add the following code:
+
+```C#
+using NewRelic.MAUI.Plugin;
+...
+    public App ()
+    {
+      InitializeComponent();
+
+      MainPage = new AppShell();
+      
+      CrossNewRelic.Current.HandleUncaughtException();
+      CrossNewRelic.Current.TrackShellNavigatedEvents();
+
+      // Set optional agent configuration
+      // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
+      // AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");
+
+      if (DeviceInfo.Current.Platform == DevicePlatform.Android)
+      {
+        CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
+        // Start with optional agent configuration 
+        // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>", agentConfig);
+      } else if (DeviceInfo.Current.Platform == DevicePlatform.iOS)
+      {
+        CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
+        // Start with optional agent configuration 
+        // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>", agentConfig);
+      }
+    }
+
+```
+
+2. In the config file above, replace `<APP-TOKEN-HERE>` with your custom application token(s) (copied from [Step 3]. If you're monitoring both Android-native and iOS-native apps, you'll need to add two different application tokens.
```

Comment:
I feel calling it `android-native` and `ios-native` on both the .NET MAUI and Xamarin pages is confusing. Maybe a better description would be `Xamarin.Android` & `Xamarin.iOS` for Xamarin and `.NET Android` & `.NET iOS`